### PR TITLE
Update to ruby 2.4

### PR DESCRIPTION
**Pull request by Jean85**
Last deployment on develop failed, due to the fact that was running on Ruby 2.4, and the Gemfile.lock expected 2.1

I attempted a `bundle update` after requiring Ruby 2.4 and it seems to work. Only downside seems to be this post-install message:
```
Post-install message from i18n:

HEADS UP! i18n 1.1 changed fallbacks to exclude default locale.
But that may break your application.

Please check your Rails app for 'config.i18n.fallbacks = true'.
If you're using I18n (>= 1.1.0) and Rails (< 5.2.2), this should be
'config.i18n.fallbacks = [I18n.default_locale]'.
If not, fallbacks will be broken in your app by I18n 1.1.x.

For more info see:
https://github.com/svenfuchs/i18n/releases/tag/v1.1.0

Post-install message from sass:

Ruby Sass is deprecated and will be unmaintained as of 26 March 2019.

* If you use Sass as a command-line tool, we recommend using Dart Sass, the new
  primary implementation: https://sass-lang.com/install

* If you use Sass as a plug-in for a Ruby web framework, we recommend using the
  sassc gem: https://github.com/sass/sassc-ruby#readme

* For more details, please refer to the Sass blog:
  http://sass.logdown.com/posts/7081811
```

### CODE REVIEW

**Comment by Jean85** [comment]
Damng, build seems to fail anyway...

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.1
+2.4
```
Comment (stefanotorresi) [inline .ruby-version]: I think we should downgrade this to 2.3 at this point.

File: Dockerfile [hunk 2]
```diff
@@ -1,11 +1,11 @@
-FROM ruby:2.4-alpine3.6 as ruby
-FROM php:7.2-alpine3.6 as dev
+FROM ruby:2.3-alpine3.8 as ruby
+FROM php:7.3-alpine3.8 as dev
```
Comment (stefanotorresi) [inline Dockerfile]: why don't we bump to alpine 3.9 while we're at it?

File: .ruby-version (hunk 1 above)
Comment (Jean85) [reply .ruby-version]: In truth, it seems that this file doesn't affect anything in the build. I would prefer remove it at this point.

File: Dockerfile (hunk 2 above)
Comment (Jean85) [reply Dockerfile]: Ruby doesn't have it. I would prefer to remain aligned in that respect between the two images.

**Comment by Jean85** [comment]
@stefanotorresi I've applied my changes. If everything is good, I would like to merge to develop.

**Pull request merged**